x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/ml/update_trained_model_deployment.yml

```diff
@@ -0,0 +1,10 @@
+---
+"Test with unknown model id":
+  - do:
+      catch: missing
```

Comment:
Since the only YAML test throws an exception the clients flight recorder won't be able to record the success response format.

Would it be possible to incorporate an update into the middle of one of the tests in `3rd_party_deployment.yml` that starts a deployment?

Reply:
Definitely. I'll add one.

Reply:
I couldn't see where this was added.

Reply:
Sorry, I forgot to pick that commit from another branch where I added it. It should be present now in `3rd_party_deployment.yml`.